src/exportable/ExportableConfiguration.ts

```diff
@@ -1,6 +1,7 @@
 import YAML from 'yaml';
 import { fshtypes } from 'fsh-sushi';
 import { Exportable } from '.';
+import { ImplementationGuideDependsOn } from 'fsh-sushi/dist/fhirtypes';
```

Comment:
We've been trying to avoid importing from `fsh-sushi/dist/**` and preferring imports directly from `fsh-sushi`.  So in this case you would do:
```ts
import fhirtypes from 'fsh-sushi';

// ... then to use ImplementationGuideDependsOn later:
this.config.dependencies.forEach((dependency: fhirtypes.ImplementationGuideDependsOn) => {
```

It's a trade-off for sure...  One could argue that one approach is better than the other, but... for now, this is the way we're doing it!